lib/validateEmail.js

```diff
@@ -0,0 +1,11 @@
+export default function validateEmail(email) {
+  let emailAddress = email;
+  let regexEmail = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/;
+  if (emailAddress.match(regexEmail)) {
```

Comment:
this doesn't look right.
should be checking if email exists first and then the `match` test.
Also this can be simplified

Reply:
changed in commit